src/helm/tasks/docker.yml

```diff
@@ -0,0 +1,27 @@
+---
+  version: 1
+  name: docker
+  schedule:
+    type: "simple"
+    interval: "10s"
+    deadline: "10s"
+  max-failures: -1
+  workflow:
+    collect:
+      metrics:
+        /intel/procfs/*: {}
```

Comment:
Can we constrain list of collected metrics to only those needed to be reported:

- file/cpu/memory utilization for hosts
- File/cpu/memory utilization for containers
- Disk utilization for for persistent disks